src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/TasksController.kt

```diff
@@ -107,27 +111,47 @@ class TasksController(
     return@runBlocking ResponseEntity.ok(tasks)
   }
 
-  override fun tasksTaskTypeGet(taskType: String): ResponseEntity<List<Task>> = runBlocking {
+  override fun tasksTaskTypeGet(
+    taskType: String,
+    page: Int?,
+    sortDirection: SortDirection?,
+  ): ResponseEntity<List<Task>> = runBlocking {
     val user = userService.getUserForRequest()
     val tasks = mutableListOf<Task>()
     val type = enumConverterFactory.getConverter(TaskType::class.java).convert(
       taskType.kebabCaseToPascalCase(),
     ) ?: throw NotFoundProblem(taskType, "TaskType")
 
+    var paginationMetaData: PaginationMetadata? = null
+
     if (user.hasRole(UserRole.CAS1_MATCHER)) {
       when (type) {
         TaskType.placementApplication -> {
+          val (placementApplications, metaData) =
+            placementApplicationService.getVisiblePlacementApplicationsForUser(
+              user,
+              page,
+              sortDirection,
+            )
+          paginationMetaData = metaData
```

Comment:
It's a bit annoying that we can't assign `metaData` and use it further down, rather than having to reassign this variable, but I understand why you've done it like that.